test/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ target_compile_definitions(test-sfml-system PRIVATE
     EXPECTED_SFML_VERSION_MAJOR=${SFML_VERSION_MAJOR}
     EXPECTED_SFML_VERSION_MINOR=${SFML_VERSION_MINOR}
     EXPECTED_SFML_VERSION_PATCH=${SFML_VERSION_PATCH}
-    EXPECTED_SFML_VERSION_IS_RELEASE=$<IF:$<BOOL:${VERSION_IS_RELEASE}>,true,false>
+    EXPECTED_SFML_VERSION_IS_RELEASE=${VERSION_IS_RELEASE_bool}
```

Comment:
Would the tests be marginally less circular if the original generator expression logic was kept?

Reply:
You still need this variable to configure the header with a C++ boolean value and the genex was doing exactly that anyway.

Reply:
Right, but if we keep the current implementation, we're also testing the logic that initializes `VERSION_IS_RELEASE_bool`.